uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/GridWidget.java

```diff
@@ -99,6 +99,16 @@
      */
     boolean isSelected();
 
+    /**
+     * Show context menu of a cell at coordinates 'uiRowIndex' and 'uiColumnIndex'.
+     * If the provided Canvas coordinate does not resolve to a cell in the Grid no operation if performed.
```

Comment:
A minor typo - I think it should be "is performed"

Reply:
Sure, well spotted, thank you.